Extension/package.json

```diff
@@ -45,7 +45,36 @@
     "virtualWorkspaces": false
   },
   "activationEvents": [
-    "*"
+    "onLanguage:c",
```

Comment:
The value in `activationEvents` should be reverted back to `"*"`

Reply:
Ugh, missed that in the last iteration. Fixed now.